rabbit-escape-engine/src/rabbitescape/engine/behaviours/Walking.java

```diff
@@ -274,6 +274,7 @@ public boolean behave( World world, Rabbit rabbit, State state )
                 return true;
             }
             case RABBIT_TURNING_LEFT_TO_RIGHT:
+                rabbit.onSlope = false; // Intentional fall-through
             case RABBIT_TURNING_LEFT_TO_RIGHT_RISING:
```

Comment:
onSlope has been implicated in several bugs. 

Is there some problem with bridge building if it is reset each step without a lot of care?